tbx/static_src/sass/components/_link.scss

```diff
@@ -0,0 +1,35 @@
+@use "config" as *;
+
+.link {
+    $root: &;
+    @include link-styles();
+
+    &--with-arrow {
+        display: inline-flex;
+        align-items: center;
+        column-gap: $spacer-mini;
+
+        #{$root}__icon {
+            transition: transform $transition-quick, fill $transition-quick;
+            transform: translate3d(0, 0, 0);
+
+            // Hide the arrow in high-contrast mode as
+            // there's already an underline on hover/focus
+            @include high-contrast-mode() {
+                display: none;
+            }
+        }
+
+        &:focus {
+            padding-right: $link-transition-gap;
```

Comment:
This is needed so that the focus ring surrounds the arrow

Reply:
Added in https://github.com/torchbox/torchbox.com/pull/16/commits/ae67a428de8e607be7fdf482a7fcf34985927e5e